Makefile

```diff
@@ -1,13 +1,30 @@
 EXTVERSION   = 0.0.1
-REGRESS      = 01_general 02_partitioning
-REGRESS_OPTS = --inputdir=test
+PGFILEDESC   = "pg_anonymize - perform data anonymization transparently on the database"
 
-PG_CONFIG = pg_config
+VPATH        = "$(CURDIR)/regress"
```

Comment:
I'm a bit worried of using a hack that relies on VPATH to replace a hack that relies on having --inputdir fed twice to pg_regress.

I know that the --inputdir=test is a bit hacky, but I've been using that for about a decade without any breakage, so it's probably good enough without any sign that the getopt rules will change anytime soon.

IMHO, if we're going to remove the --inputdir hack, we should just rely on pg_regress default behavior and accept having a sql/ directory in the root directory.  I'm not a fan of that as it feels a bit counterintuitive for an extension to have sql/ containing test and not the SQL part of the extension, but that's how postgres works and we shouldn't end up to a point where we have so many SQL extension files that we would need another specific directory for that.  Would that be ok for you?

Reply:
Fair enough :)

I took another stab at it. This time round, I use the --input-dir as before, adding --output-dir to it. It is all under the same roof, i.e. EXTRA_REGRESS_OPTS simply because this variable is a slightly easier to trace in postgres's codepath. 

I still think that having the input and output under the same parent directory makes sense and I agree completely with you that having a bare `sql` directory in parent for tests is counter intuitive. I chose to stick the naming to `regress` because it matches the targets and variables around, but I would not mind to stick to `test` if you would prefer.

Thank you for the review and comments!